### PR TITLE
cyberpunk-neon: init at unstable-2021-02-02

### PR DESCRIPTION
**Pull request by rpgwaiter**
###### Motivation for this change
I wanted to add the desktop theme I use to nixpkgs

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)

   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
   
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/themes/cyberpunk-neon/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+    pname = "cyberpunk-neon";
+    version = "b4b293c";
```
Comment (Pacman99) [inline pkgs/data/themes/cyberpunk-neon/default.nix:5]:
```suggestion
    version = "2021-02-02";
```
https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

File: pkgs/data/themes/cyberpunk-neon/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        # terminal
+        mkdir -p $out/share/konsole
+        cp terminal/konsole/cyberpunk-neon.colorscheme $out/share/konsole
+        mkdir -p $out/share/tilix/schemes
+        cp terminal/tilix/cyberpunk-neon.json $out/share/tilix/schemes
+    '';
+
+    meta = with lib; {
+        description = "Theme for GTK, plasma, konsole and tilix ";
```
Comment (Pacman99) [inline pkgs/data/themes/cyberpunk-neon/default.nix:34]: you could probably include the themes description from the README here

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 114522` at d9ebac56 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li><a href="https://gist.github.com/7486e900b4be117244a7f5cfc690af24">cyberpunk-neon</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/data/themes/cyberpunk-neon/default.nix:16:5:
```
   |
16 |     installPhase = ''
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/data/themes/cyberpunk-neon/default.nix (hunk 2 above)
Comment (rpgwaiter) [reply pkgs/data/themes/cyberpunk-neon/default.nix]: Went through and edited the commit with your suggestions, thanks!

**Review by Pacman99 — APPROVED** [review]

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please intend with two spaces.

**Comment by rpgwaiter** [comment]
> Please intend with two spaces.

I went through and fixed the indents, thanks.

File: pkgs/data/themes/cyberpunk-neon/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "cyberpunk-neon";
+  version = "2021-02-02";
+
+  src = fetchFromGitHub {
+    owner = "Roboron3042";
+    repo = "Cyberpunk-Neon";
+    rev = version;
```
Comment (SuperSandro2000) [inline pkgs/data/themes/cyberpunk-neon/default.nix:10]: Releases got removed upstream. Can you fetch a commit instead and change the version number to unstable-XXXX-XX-XX?

Comment (rpgwaiter) [reply pkgs/data/themes/cyberpunk-neon/default.nix]: Done, thanks.

File: pkgs/data/themes/cyberpunk-neon/default.nix [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+    # terminal
+    mkdir -p $out/share/konsole
+    cp terminal/konsole/cyberpunk-neon.colorscheme $out/share/konsole
+    mkdir -p $out/share/tilix/schemes
+    cp terminal/tilix/cyberpunk-neon.json $out/share/tilix/schemes
+  '';
+
+  meta = with lib; {
+    description = "Themes for KDE Plasma, GTK, Telegram, Tilix, Vim, Zim and more.";
```
Comment (SuperSandro2000) [inline pkgs/data/themes/cyberpunk-neon/default.nix:34]:
```suggestion
    description = "Themes for KDE Plasma, GTK, Telegram, Tilix, Vim, Zim and more";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by AndersonTorres** [comment]
Superseded by https://github.com/NixOS/nixpkgs/pull/331410

**Pull request closed without merging**